code/game/objects/structures/ladders.dm

```diff
@@ -24,8 +24,13 @@
 		link_up(up)
 	if(down)
 		link_down(down)
-
-	return INITIALIZE_HINT_LATELOAD
+	var/turf/open/openspace/downturf = get_step_multiz(get_turf(src), DOWN)
+	var/turf/open/openspace/upturf = get_step_multiz(get_turf(src), UP)
+	if(!downturf && !upturf) //we only add markers to multi-z ladders, sorry Sulaco
```

Comment:
Couldn't you just use up and down vars here instead? Would make it work on sulaco too